code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/ldap/LDAPIdentityProvider.java

```diff
@@ -194,6 +194,7 @@ protected Identity attributesToIdentity(LdapName dn){
             String accountName;
             String externalId = dn.toString();
             String login;
+            Boolean user = false;
```

Comment:
@cjellick ah yes, hadn't checked the PR again after updating it and this didn't show any errors in eclipse so didn't realize